bundles/org.openhab.binding.webexteams/src/main/java/org/openhab/binding/webexteams/internal/api/WebexTeamsApi.java

```diff
@@ -128,30 +123,29 @@ private <I, O> O doRequest(URI url, HttpMethod method, String authToken, Class<O
                 req.header("Content-type", "application/json");
             }
 
-            InputStreamResponseListener listener = new InputStreamResponseListener();
-            req.send(listener);
+            ContentResponse response = req.send();
 
-            // Wait for the response headers to arrive
-            Response response = listener.get(5, TimeUnit.SECONDS);
             logger.debug("Response: {} - {}", response.getStatus(), response.getReason());
 
             if (response.getStatus() == HttpStatus.UNAUTHORIZED_401) {
+                response.abort(new Exception(response.getReason()));
```

Comment:
No more needed now that you are using a synchronous HTTP call.

Reply:
Removed.